e2e/cypress/integration/interactive_dialog/scrollable_spec.js

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/**
+* Note: This test requires webhook server running. Initiate `npm run start:webhook` to start.
+*/
+
+const webhookUtils = require('../../../utils/webhook_utils');
+
+let createdCommand;
+let userAndChannelDialog;
+
+describe('Interactive Dialog', () => {
+    before(() => {
+        // # Check webhook
+        cy.request(Cypress.env().webhookBaseUrl).then((response) => {
+            expect(response.status === 200, 'This test requires webhook server running. Initiate `npm run start:webhook` to start.');
+        });
```

Comment:
If webhook server is not running, this terminates Cypress process and generates hard to read error.  I opened [PR-5012](https://github.com/mattermost/mattermost-webapp/pull/5012/files#diff-e28127a472a0ddc91f9bf050e02a0556R81) to address this and I suggest to wait for it to merge so you could use `cy. requireWebhookServer()`

Reply:
fyi, said PR has been merged.  Please update and we're good to merge.